### DISCUSSION TITLE
Webinterface doesnt look right

### DISCUSSION THREAD

**Discussion by ilu2k21** [jomjol/AI-on-the-edge-device#1263] [Q&A] [answered]
Initial setup was ok, but afterward the webinterface is kind off.
![Bildschirmfoto 2022-11-03 um 14 52 41](https://user-images.githubusercontent.com/78492525/199740201-7dd87db3-813c-42d6-b694-0499a4938c9a.png)

Makes it hard to change config.

Any hints?

**Comment by pos-ei-don** [comment]
Hit F12 and See on the network Tab, if a css file or anything like this cannot be recived by the Webserver.

**Reply by ilu2k21** [reply]
there are 4 files that couldnt be loaded:
rssi.html
cputemp.html
alg_roi.jpg
cputemp.html

everything else seems ok

**Reply by caco3** [reply]
refresh the page with F5 then it should be fine.

Also: is there a reason you are not using the latest version?

**Comment by ilu2k21** [comment, marked as answer]
It works after updating to newest version. Dont know why i started with the older version
